recipes/bshoshany-thread-pool/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,815 @@
+/**
```

Comment:
Keep it very short, a single method call is enough. The idea is testing the packaged libraries, not run unit tests or examples.